tests/test_blscache.py

```diff
@@ -0,0 +1,58 @@
+from chia_rs import G1Element, PrivateKey, AugSchemeMPL, G2Element, BLSCache
+from chia.util.ints import uint64
+from chia.types.blockchain_format.sized_bytes import bytes48
```

Comment:
you should import these from `chia_rs`.
```
from chia_rs.sized_bytes import bytes48
```

Reply:
After moving to PublicKey we no longer need bytes48 at all - it's removed now